cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,261 @@
+    cy.get('.rt-tbody .rt-td').should('contain',worker.email);
+    cy.get('.rt-tbody .rt-td').should('contain',worker.age.toString());
+   
+  });
+
+  it('should validate data in worker row after creating worker', () => {
+    
+    const worker = generateWorker();
+
+    cy.get('#addNewRecordButton').click();
+
+    cy.get('#firstName').type(worker.firstName);
+    cy.get('#lastName').type(worker.lastName);
+    cy.get('#userEmail').type(worker.email);
+    cy.get('#age').type(worker.age);
+    cy.get('#salary').type('15000');
+    cy.get('#department').type('Sales');
+
+    cy.get('#submit').click();
+
+ 
+    cy.get('.rt-td').should('contain', worker.firstName);
+    cy.get('.rt-td').should('contain', worker.lastName);
+    cy.get('.rt-td').should('contain', worker.email);
+    cy.get('.rt-td').should('contain', worker.age.toString());
+    cy.get('#salary').should('have.value', '15000');
+    cy.get('#department').should('have.value', 'Sales');
+  });
+
+  it('should check search by all column values', () => {
+    
+    const worker = generateWorker();
+
+    cy.get('#addNewRecordButton').click();
+
+    cy.get('#firstName').type(worker.firstName);
+    cy.get('#lastName').type(worker.lastName);
+    cy.get('#userEmail').type(worker.email);
+    cy.get('#age').type(worker.age);
+    cy.get('#salary').type('12000');
+    cy.get('#department').type('Sales');
+
+    cy.get('#submit').click();
+    
+
+    cy.get('.rt-td').should('contain', worker.firstName);
+    cy.get('.rt-td').should('contain', worker.lastName);
+    cy.get('.rt-td').should('contain', worker.email);
+    cy.get('.rt-td').should('contain', worker.age.toString());
+    //cy.get('#salary').should('have.value', '12000');
+    //cy.get('#department').should('have.value', 'Sales');
+
+    cy.get('input#searchBox').clear().type(worker.firstName);
+    cy.get('.rt-td').should('contain', worker.firstName);
+    cy.get('.rt-td').should('contain', worker.lastName);
+    cy.get('.rt-td').should('contain', worker.email);
+    cy.get('.rt-td').should('contain', worker.age.toString());
+    cy.wait(500); 
+
+    cy.get('input#searchBox').clear().type(worker.lastName);  
+    cy.get('.rt-td').should('contain', worker.firstName);
+    cy.get('.rt-td').should('contain', worker.lastName);
+    cy.get('.rt-td').should('contain', worker.email);
+    cy.get('.rt-td').should('contain', worker.age.toString());
+
+    cy.get('input#searchBox').clear().type(worker.email);  
+    cy.get('.rt-td').should('contain', worker.firstName);
+    cy.get('.rt-td').should('contain', worker.lastName);
+    cy.get('.rt-td').should('contain', worker.email);
+    cy.get('.rt-td').should('contain', worker.age.toString());
+
+    cy.get('input#searchBox').clear().type(worker.age);  
+    cy.get('.rt-td').should('contain', worker.firstName);
+    cy.get('.rt-td').should('contain', worker.lastName);
+    cy.get('.rt-td').should('contain', worker.email);
+    cy.get('.rt-td').should('contain', worker.age.toString());
```

Comment:
Missing newline at end of file. It's a common convention to put a newline at the end of a file. Some tools might not handle the last line correctly if it doesn't end with a newline.